Description: Patch config.{sub,guess} to call their up-to-date versions.
Author: أحمد المحمودي (Ahmed El-Mahmoudy) <aelmahmoudy@sabily.org>
--- a/config.guess
+++ b/config.guess
@@ -1,4 +1,8 @@
 #! /bin/sh
+if [ -x /usr/share/misc/config.guess ]; then
+  exec /usr/share/misc/config.guess "$@"
+fi
+
 # Attempt to guess a canonical system name.
 #   Copyright (C) 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
 #   2000, 2001, 2002, 2003, 2004, 2005 Free Software Foundation, Inc.
--- a/config.sub
+++ b/config.sub
@@ -1,4 +1,8 @@
 #! /bin/sh
+if [ -x /usr/share/misc/config.sub ]; then
+  exec /usr/share/misc/config.sub "$@"
+fi
+
 # Configuration validation subroutine script.
 #   Copyright (C) 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
 #   2000, 2001, 2002, 2003, 2004, 2005 Free Software Foundation, Inc.
